server/lib/src/server/migrations.rs

```diff
@@ -594,6 +595,8 @@ impl<'a> QueryServerWriteTransaction<'a> {
         // First, check the system_info object. This stores some server information
         // and details. It's a pretty const thing. Also check anonymous, important to many
         // concepts.
+
+        // The below only check if the UUID of the classes exists (Should exists from migrations)
```

Comment:
What does this comment mean?

Reply:
It was a personal note to keep track of code. Removing it